src/components/Results.tsx

```diff
@@ -31,93 +30,63 @@ export const Results: React.FC<{
     [facilities, facilityIds]
   );
 
-  const handleResults: () => Promise<LatLngTuple | undefined> = useCallback(async () => {
-    logger.info('requst results');
+  const handleResults = useCallback(async () => {
+    logger.info('Init results fetch');
     setLoading(true);
     setError(undefined);
 
     try {
       if (searchParams === undefined) {
         throw new Error('searchParams must be provided');
```

Comment:
It is always better to validate incoming data. You can implement validation using JSON schema or create a simple method for it but using JSON schema is a more standardized and flexible way.

Reply:
done